### PR TITLE
Link generation

### PR DESCRIPTION
**Pull request by sanjay611080**
This pull request includes the following changes:

- **Assessments Displayed in List**  
  The assessments are now displayed in a list format. We initially tried to use a common table to display the assessment data, but encountered some issues. We will resolve this soon.

  Assessments List

- **User Details in Filtered and Selected Sections**  
  The user's details are displayed in the filtered section, and after selecting a particular user, they will appear in the selected section.

  User Details

- **Toggle Issues Fixed**  
  The toggle issues have been fixed and are now working properly.

  Toggle Working

  Another Toggle

- **Link Sent to Users and Stored in Database**  
  When the link is sent to users, it will be stored in the database, and the records will be shown here.

  Link Sent

- **Expired Link Access Page**  
  If the user tries to access an expired link, this page will be displayed.

  Expired Link

- **Link Already Accessed Page**  
  This page will appear if the user has already accessed the link and tries to access it again.

  Link Already Accessed

---

Now, we'll start working on link status and shortening the generated link before saving it to the database and replace the assessment list with the common table.

### CODE REVIEW

File: src/app/components/linkGeneration/Pages/assessment-records/assessment-records.component.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+@Component({
+  selector: 'app-assessment-records',
+  standalone: true,
+  imports: [FormsModule,RouterLink],
+  templateUrl: './assessment-records.component.html',
+  styleUrl: './assessment-records.component.css'
+})
+export class AssessmentRecordsComponent implements OnInit {
+[x: string]: any;
```
Comment (goelp30) [inline src/app/components/linkGeneration/Pages/assessment-records/assessment-records.component.ts:15]: What is this x:string doing

File: src/app/enums/TableName.ts [hunk 2]
```diff
@@ -1,3 +1,7 @@
 export const enum TableNames {
-    Subject = 'subject'
-}
\ No newline at end of file
+    Employee = 'employee',
+    Candidate = 'candidate',
+    Subject = "Subject"
```
Comment (goelp30) [inline src/app/enums/TableName.ts:4]: Camel case it

File: src/app/components/linkGeneration/Pages/assessment-records/assessment-records.component.ts (hunk 1 above)
Comment (sanjay611080) [reply src/app/components/linkGeneration/Pages/assessment-records/assessment-records.component.ts]: We were using it to allow dynamic properties to be added to the AssessmentRecordsComponent instance. But we forget to remove it

File: src/app/enums/TableName.ts (hunk 2 above)
Comment (sanjay611080) [reply src/app/enums/TableName.ts]: Camel case done

**Review by goelp30 — APPROVED** [review]

**Review by Krishan-Chamoli — APPROVED** [review]

**Pull request merged**